patches/tModLoader/Terraria/ModLoader/Core/ModOrganizer.cs

```diff
@@ -535,40 +535,34 @@ private enum SearchFolders { }
 			if (tmods.Length <= 3)
 				return;
 
-			string location = FindOldest(repo);
-			if (location.EndsWith(".tmod"))
-				File.Delete(location);
-			else
-				Directory.Delete(location, true);
-		}
-
-		internal static string FindOldest(string repo) {
-			string[] tmods = Directory.GetFiles(repo, "*.tmod", SearchOption.AllDirectories);
-			if (tmods.Length == 1)
-				return tmods[0];
+			var compareVersion = DecreaseDateVersion(BuildInfo.ltsVersion);
```

Comment:
Sure, keep all versions since the last `LTS`. Seems like a good start.

Reply:
To be precise - I'm keeping LTS minus 1 for rollover safety. 

For example, for Preview = 2022.9
LTS = 2022.8
LTS minus 1 = 2022.7

Only problem with this is if we ran two back to back 2+ month preview. Wouldn't be guaranteed for rollback to last LTS on player base.